### PR TITLE
New UI character burn

### PR DESCRIPTION
**Pull request by Spintacle123**
PR includes:
 - Added New Icon Elements
 - Added new ui for Character Burn
 - modify the ui for Garrison
 - Added new ui for Character Sub Menu
 - optimized the mobile view

### CODE REVIEW

**Comment by Sellie96** [comment]
- Garrison button can be clicked on and off leaving a weird bug. Can't be done on the other two buttons

- Screen width is over 100% because of some weird padding / margin going on with the character and garrison pages

- Bug with the character burning Filters, text goes over other text?

- Loading Characters on the char burning page takes a long time after a refresh.

- I'm unsure about the xp bar on the characters, I don't think it's clear that it is xp till next level on the right there.

- Characters aren't updated after one is burnt. The sidebar does this if you're looking for how it is done. Also if the selected character is burnt you get some weird interface bugs

**Comment by Spintacle123** [comment]
> * Garrison button can be clicked on and off leaving a weird bug. Can't be done on the other two buttons
> * Screen width is over 100% because of some weird padding / margin going on with the character and garrison pages
> * Bug with the character burning Filters, text goes over other text?
> * Loading Characters on the char burning page takes a long time after a refresh.
> * I'm unsure about the xp bar on the characters, I don't think it's clear that it is xp till next level on the right there. (It's also an incorrect value)
> * Characters aren't updated after one is burnt. The sidebar does this if you're looking for how it is done. Also if the selected character is burnt you get some weird interface bugs

can you send the screenshot of the no. 3 issue?  cant replicate it.

File: frontend/src/views/Character.vue [hunk 1]
```diff
@@ -377,12 +392,24 @@ export default Vue.extend({
       }
       this.isUpgrading = false;
     },
+    async selectAll(){
+      await this.computeSoul();
+      await this.updateBurnCost();
+    },
+    computeSoul(){
+      this.totalSoul = 0;
```
Comment (kevinunger) [inline frontend/src/views/Character.vue:400]: `computeSoul()` can be a computed method

**Comment by kevinunger** [comment]
image
Burn Btn isn't properly positioned. This is more noticable on wider screens:
image

**Review by Sellie96 — APPROVED** [review]

**Pull request merged**